run.sh

```diff
@@ -1,67 +0,0 @@
-#!/bin/bash
-#
-# Miscellaneous scripts.
```

Comment:
It's useful for publishing to github pages, which we are using.  Please restore the file.